migrations/900000000000024_git_remotes_syncer.up.sql

```diff
@@ -0,0 +1,18 @@
+BEGIN;
+
+INSERT INTO mergestat.repo_sync_types (type, description, short_name) VALUES ('GIT_REMOTES', 'Retrieves the remotes of a git repo. Only relevant for local, on-disk repos.', 'Git Remotes') ON CONFLICT DO NOTHING;
+
+CREATE TABLE IF NOT EXISTS git_remotes (
+    repo_id uuid PRIMARY KEY,
+    name text NOT NULL,
+    url text NOT NULL,
+    _mergestat_synced_at timestamptz NOT NULL DEFAULT now()
```

Comment:
Semantically is the same but we have been using timestamp with time zone

Reply:
good call - I blame copilot 😄